feature-libs/customer-ticketing/components/customer-ticketing-list/customer-ticketing-list.component.ts

```diff
@@ -130,23 +130,25 @@ export class CustomerTicketingListComponent implements OnInit {
   }
 
   changeSortCode(sortCode: string): void {
-    // const event: { sortCode: string; currentPage: number } = {
-    //   sortCode,
-    //   currentPage: 0,
-    // };
     this.sortType = sortCode;
-    // this.fetchTicketList(event);
+    this.pageChange(0);
   }
 
   pageChange(page: number): void {
-    // const event: { sortCode: string; currentPage: number } = {
-    //   sortCode: this.sortType,
-    //   currentPage: page,
-    // };
-    console.log('pageChange', page);
+    this.createTicketListEvent(this.sortType, page);
     // this.fetchTicketList(event);
   }
 
+  createTicketListEvent(
```

Comment:
```suggestion
  createTicketListSortEvent(
```

Reply:
This event is not only for sorting but also for changing pages so I prefer to leave the name as it is